docs/spec_draft.md

```diff
@@ -348,6 +349,93 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.batch_norm_training
+
+### Semantics
+
+Computes mean and variance across all dimensions except for the `feature_index`
+dimension and normalizes the `operand` tensor producing `output`, `batch_mean`
+and `batch_var` tensors. More formally, this operation can be expressed as a
+decomposition to existing StableHLO operations using Python-like syntax as
+follows:
+
+```python
+def compute_mean(operand, feature_index):
+  (sum,) = reduce(
```

Comment:
I had first written sum then changed it to aggregate because it is reserved in python, also syntax highlighting makes it blue

Reply:
Let's keep `sum` for the sake of clarity. Calling variables `sum` is not Pythonic, but we only promised "Python-like syntax" above. Also, when Markdown is rendered, the highlighting is gone, which is nice.